uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -43,6 +45,47 @@
         }
     }
 
+   .logout-launcher,.notification-icon,.portal-greeting,.login-launcher{
```

Comment:
And here we get a close look at one of the ways in which Respondr & regions are challenging/confusing (sorry):  we hope not to have styles defined within `media/skins/respondr/common` make assumptions about what portlets are in which regions (though we're probably violating that hope, already, in many places).

In a perfect world you'll be able to sprinkle portlets any way you like into regions, and it all just "works" and looks good.  In the real world, I tend to do this kind of fine-tuning inside `media/skins/respondr/{customSkin}/less`.